cloud/aws/modules/secrets_manager/outputs.tf

```diff
@@ -12,3 +12,19 @@ output "database_password" {
   value     = data.aws_secretsmanager_secret_version.postgres_password_version.secret_string
   sensitive = true
 }
+
+output "oidc_client_id" {
+  value     = data.aws_secretsmanager_secret_version.oidc_client_id_version.secret_string
+  sensitive = true
+}
+
+output "oidc_secret" {
```

Comment:
[Here and below] For consistency, this is "oidc_client_secret"

https://www.oauth.com/oauth2-servers/client-registration/client-id-secret/